src/components/ProfilePage/AgeMessage.jsx

```diff
@@ -1,16 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { UncontrolledAlert } from 'reactstrap';
+import { FormattedMessage } from 'react-intl';
 
 function AgeMessage({ accountURL }) {
   return (
     <UncontrolledAlert color="info">
-      <h6>Your profile cannot be shared.</h6>
-      <p>
-        To share your profile with other edX learners,
-        you must confirm that you are over the age of 13.
-      </p>
-      <a href={accountURL}>Set your date of birth</a>
+      <FormattedMessage
+        id="profile.age.headline"
+        defaultMessage="Your profile cannot be shared."
```

Comment:
Are we intentionally not telling them why their profile cannot be shared?

Reply:
Yup, that's in the next line.

Reply:
:) Oops.  Thanks.  I didn't read the end of the details.  Note: I like "header" better than "headline" if we were to document a best practice, but no change needed.